src/main/java/com/azure/cosmos/examples/autoscaledatabasecrud/sync/AutoscaleDatabaseCRUDQuickstart.java

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.examples.autoscaledatabasecrud.sync;
+
+import com.azure.cosmos.ConsistencyLevel;
+import com.azure.cosmos.CosmosClient;
+import com.azure.cosmos.CosmosClientBuilder;
+import com.azure.cosmos.CosmosDatabase;
+import com.azure.cosmos.CosmosPagedIterable;
+import com.azure.cosmos.examples.changefeed.SampleChangeFeedProcessor;
+import com.azure.cosmos.examples.common.AccountSettings;
+import com.azure.cosmos.models.CosmosDatabaseProperties;
+import com.azure.cosmos.models.CosmosDatabaseRequestOptions;
+import com.azure.cosmos.models.CosmosDatabaseResponse;
+import com.azure.cosmos.models.FeedOptions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AutoscaleDatabaseCRUDQuickstart {
+
+    private CosmosClient client;
+
+    private final String databaseName = "AzureSampleFamilyDB";
+
+    private CosmosDatabase database;
+
+    protected static Logger logger = LoggerFactory.getLogger(SampleChangeFeedProcessor.class.getSimpleName());
+
+    public void close() {
+        client.close();
+    }
+
+    /**
+     * Sample to demonstrate the following AUTOSCALE database CRUD operations:
+     * -Create
+     * -Read by ID
+     * -Read all
+     * -Delete
+     */
+    public static void main(String[] args) {
+        AutoscaleDatabaseCRUDQuickstart p = new AutoscaleDatabaseCRUDQuickstart();
+
+        try {
+            logger.info("Starting SYNC main");
+            p.autoscaleDatabaseCRUDDemo();
+            logger.info("Demo complete, please hold while resources are released");
+        } catch (Exception e) {
+            e.printStackTrace();
+            logger.error(String.format("Cosmos getStarted failed with %s", e));
```

Comment:
don't mix STring.format with slf4j logger
see this for how to do logging for error using slf4j
http://www.slf4j.org/faq.html#paramException

```suggestion
            logger.error("Cosmos getStarted failed", e);
```